Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3885,6 +3885,16 @@ def gen2022HiMix(fragment,howMuch):
                          '--datatier':'NANOAODSIM',
                          '--eventcontent':'NANOEDMAODSIM',
                          '--filein':'file:step3_inMINIAODSIM.root'}
+
+steps['HCALNano']={'-s':'RAW2DIGI,RECO,USER:DPGAnalysis/HcalNanoAOD/hcalNano_cff.hcalNanoTask',
```

Comment:
Is this by intention to run Nano out of RECO directly? How actually will work in real situation, run with processing (prompt, rereco) only?

Reply:
Hi @srimanob - I'm not 100% sure I understand the question, but we intend to run this Hcal NANO over RAW for a few AlCA workflows. I think it would also work on RECO, but I'm not aware of such a use case.